.github/workflows/tests-via-crave.yml

```
@@ -0,0 +1,38 @@
+name: Solr Tests
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  test:
+    name: Run Solr Tests using Crave.io resources
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Setup
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'temurin'
+        java-version: 11
+        java-package: jdk
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - uses: actions/cache@v2
+      with:
+        path: |
+          ~/.gradle/caches
+        key: ${{ runner.os }}-gradle-solrj-${{ hashFiles('versions.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gradle-solrj-
+          ${{ runner.os }}-gradle-
+    - name: Get the Crave binary
+      run: curl -s https://raw.githubusercontent.com/accupara/crave/master/get_crave.sh | bash -s --
+    - name: Initialize gradle settings
+      run: ./crave run -- ./gradlew localSettings
+    - name: Test the Solr Package
+      run: ./crave run -- './gradlew --max-workers=`nproc` test'
```

```suggestion
      run: ./crave run -- './gradlew --max-workers=`nproc` -Ptests.jvms=48 test'
```

I did some exploration and found that it's not enough to set the workers in this way as the "tests.jvms" will default to the value in gradle.properties which is again capped at 12.  When I used 48 workers & test JVMs on this machine, the tests finished in 5:45 which is awesome!  If you wish we could use nproc again thus 96 for both.  I did one such build and it pegged the cpu but then stalled out, probably due to misbehavior in a test or something.  I ultimately killed it.  Could just be bad luck.

---

Updated as requested.